client/src/components/SearchBar/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+// import MagnifierDisabled from "../../images/1.Icons/icon-magnifier-disabled.svg";
```

Comment:
Be careful about forgetting commented code in your repository, it's not a good practice.

Reply:
Hi Danilo! I was assuming that here you are referring to non-descriptive comments which are not helpful for me and for other people to understand the code, am I right?

Reply:
Sorry for the late answer, I was on vacation.

Actually, I was talking about this SVG's import which is commented line and you're not using it anywhere